Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -82,19 +95,13 @@ spec:
       - ip: 192.168.39.182
 ```
 
-You can use the existing `object-external.yaml` file.
-When ready the ceph-object-controller will output a message in the Operator log similar to this one:
-
-```console
-ceph-object-controller: ceph object store gateway service >running at 10.100.28.138:8080
-```
+You can use the existing `object-external.yaml` file. Even though multiple endpoints can be specified, it is recommend to use only one endpoint. This endpoint is randomly added to `configmap` of OBC and secret of the `cephobjectstoreuser`. Rook never guarnatees the randomly picked endpoint is a working one or not.
+If there are multiple endpoints, please add load balancer in front of them and use the load balancer endpoint in the `externalRgwEndpoints` list.
 
-You can now get and access the store via:
+When ready the ceph-object-controller will output a message in the operator log similar to this one:
```

Comment:
There is no status other than grep'ing the operator log? It's preferred we don't refer them to the logs unless they're troubleshooting.

Reply:
User can check the object store status and see it's connected or not. This doc was bit old